src/org/samples/basicapp/BasicApp.java

```diff
@@ -0,0 +1,60 @@
+package org.samples.basicapp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.moonlightcontroller.bal.BoxApplication;
+import org.moonlightcontroller.blocks.FromDevice;
+import org.moonlightcontroller.blocks.HeaderClassifier;
+import org.moonlightcontroller.blocks.ToDevice;
+import org.moonlightcontroller.processing.Connector;
+import org.openboxprotocol.protocol.HeaderField;
+import org.openboxprotocol.protocol.HeaderMatch;
+import org.openboxprotocol.protocol.IStatement;
+import org.openboxprotocol.protocol.OpenBoxHeaderMatch;
+import org.openboxprotocol.protocol.Priority;
+import org.openboxprotocol.protocol.Statement;
+import org.openboxprotocol.types.Port;
+
+public class BasicApp extends BoxApplication {
```

Comment:
This is just a basic app for testing and for the milestone it doesn't have any real meaning.
I'm using it to load with the registry and for experimenting. 
I thought it was a good idea that you would have it too.